modules/twinklewarn.js

```diff
@@ -110,6 +116,34 @@ Twinkle.warn.callback = function twinklewarnCallback() {
 		tooltip: 'A page can be linked within the notice, perhaps because it was a revert to said page that dispatched this notice. Leave empty for no page to be linked.'
 	});
 
+	form.append({
+		type: 'div',
+		label: '',
+		style: 'color: red',
```

Comment:
I have essentially zero experience with anti-vandalism so I wouldn't know, but is it really  wrong to issue warnings for edits that are more than 24 hours old? If the answer if anything other than an obvious yes, I'd rather the color be toned down to something less severe.

Reply:
It's actually comparatively soft to other tools: Huggle pops up an alert requiring confirmation (IIRC it also has a user preference to always skip issuing the warning entirely), and STiki has a popup as well.  As a sysop, I wouldn't say it's always wrong, but it should definitely be treated differently, so I think the red is appropriately eye catching.  The message is fairly tame.